plugins/schedule/base/lib/api/filters/base/KalturaEntryScheduleEventBaseFilter.php

```diff
@@ -8,7 +8,6 @@
 {
 	static private $map_between_objects = array
 	(
-		"templateEntryIdEqual" => "_eq_template_entry_id",
```

Comment:
This is generated code and shouldn't be changed manually.
If you have duplication, remove the other occurrence.